protocol/interop/interop-launch-sequence.md

```diff
@@ -0,0 +1,55 @@
+# Interop Launch Sequence
+
+## Purpose
+
+This document describes a proposed sequence of events that will occur when interop goes live on
+mainnet. The exact sequence is subject to change, but this document is meant to provide a scaffold
+that helps drive other design documents.
+
+## Proposed Sequence
+
+- Far before the launch of interop, for all chains, in an OPCM action per chain:
+  - An `ETHLockbox` contract is deployed for the chain.
+  - The Upgrade Controller approves the `OptimismPortal` address to use the `ETHLockbox`.
+  - Chains are upgraded to the new `OptimismPortal` implementation.
+    - Because `usingSuperRoots` is `false` by default, this will not result in any immediate impact
+      on the proving codepath that users will have to use.
+    - `OptimismPortal` will point to the `ETHLockbox`.
+  - The Upgrade Controller triggers the `migrate` function on the `OptimismPortal` which causes the
+    `OptimismPortal` to transfer all of its ETH into the `ETHLockbox`.
+- Shortly before the scheduled activation of interop on L2:
+  - The Superchain-wide pause is activated.
+    - TODO: Decide if this is really the correct way to do this, or if there's some other mechanism
+      that works well enough. We essentially need to stop users from using the old proof system
+      while the new proof system is spun up.
```

Comment:
This seems pretty excessive and suggests we haven't thought through the migration properly. The pause also doesn't prevent playing dispute games so if there's something that isn't working in the proofs system, bonds will be lost and games may resolve incorrectly requiring blacklisting before we unpause.

Reply:
Maybe another approach is to set the respected game type to a non-existent one, 404, this way withdrawals are effectively paused and no one can play dispute games